src/interface.jl

```diff
@@ -645,9 +651,27 @@ end
 
 function jacobian_conjugate! end
 @doc "$(_doc_jacobian_conjugate)"
-jacobian_conjugate!(
-    ::LieGroup, J, g, h; B::AbstractBasis=DefaultLieAlgebraOrthogonalBasis()
+function jacobian_conjugate!(
+    G::LieGroup,
+    J,
+    g,
+    h,
+    B::AbstractBasis=DefaultLieAlgebraOrthogonalBasis();
+    X=zero_vector(LieAlgebra(G)),
 )
+    n = number_of_coordinates(base_manifold(G), B)
+    𝔤 = LieAlgebra(G)
+    c = zeros(eltype(J), n)
+    for j in 1:n
+        c .= 0
+        c[j] = 1
+        get_vector!(𝔤, X, c, B)         # store ``X_j`` in X
+        diff_conjugate!(G, X, g, h, X) # compute the differential in-place
+        get_coordinates!(𝔤, c, X, B)   # compute its coordinates in B
+        J[:, j] .= c                    # store the result in J
```

Comment:
```suggestion
        get_coordinates!(𝔤, view(J, :, j), X, B)   # compute its coordinates in B and store the result in J
```
This can work in one step